datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -19,14 +19,32 @@ const defaultProps = {
 
 export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink }: Props) => {
     const entityRegistry = useEntityRegistry();
-
     const handleLogout = () => {
         isLoggedInVar(false);
         Cookies.remove(GlobalCfg.CLIENT_AUTH_COOKIE);
     };
 
     const menu = (
         <Menu>
+            <Menu.Item
```

Comment:
Since these links may not be the right links for all installations of datahub, could we make the number of links and their contents configurable? 
That way different installations can customize what they want in the dropdown.

Reply:
Ideally these should be configured via our app config file: https://github.com/linkedin/datahub/tree/master/datahub-web-react/src/conf/theme

see more here:
https://github.com/linkedin/datahub/tree/master/datahub-web-react#theming

Reply:
Please look into the updated solution, where i have added the menu configuration in theme file.